### PR TITLE
Include Tier-1 bootstrap test suites in pipeline

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- Include Tier-1 bootstrap test suites in pipeline

### CODE REVIEW

File: pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
+    Pipeline script for executing Tier 0 test suites for RH Ceph 5.0.
```
Comment (rakeshgm) [inline pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy:2]: i think this for Tier 1 right. can you change the word `Tier 0`

File: pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                            }
+                        }
+                    }
+                 }, 'Bootstrap_skip_dashboard': {
+                    stage('Bootstrap suite - Skip dashboard') {
+                        sleep(180)
+                        script {
+                            withEnv([
+                                "sutVMConf=conf/inventory/rhel-8.3-server-x86_64-medlarge.yaml",
+                                "sutConf=conf/${cephVersion}/rgw/tier1_3node_cephadm_bootstrap.yaml",
```
Comment (psathyan) [inline pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy:28]: I believe the path is incorrect... kindly check the parent folder.

File: pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                                sharedLib.runTestSuite()
+                            }
+                        }
+                    }
+                 }, 'Bootstrap_skip_monitoring_stack_orphan_initial_daemons': {
+                    stage('Bootstrap suite - skip-monitoring-stack and orphan-initial-daemons') {
+                        sleep(360)
+                        script {
+                            withEnv([
+                                "sutVMConf=conf/inventory/rhel-8.3-server-x86_64-medlarge.yaml",
```
Comment (psathyan) [inline pipeline/5/jenkinsfile-tier-1-deployment-bootstrap.groovy:41]: Likewise, the below paths don't exist

**Review by psathyan — COMMENTED** [review]
Looks good to me... once RHOS-D is online, let's get one run before we merge this PR

**Comment by sunilkumarn417** [comment]
**Test results:**
https://storage-jenkins-csb-ceph.cloud.paas.psi.redhat.com/view/RHCS%205%20QE/job/rhceph-5-tier-0/75/consoleFull

**Review by psathyan — APPROVED** [review]

**Pull request merged**